src/app/util/clipboard.cpp

```diff
@@ -190,7 +190,7 @@ Clipboard::~Clipboard()
 
 void Clipboard::setClipboardText(const std::string& text)
 {
-  if (use_native_clipboard()) {
+  if (use_native_clipboard() && App::instance()->isGui()) {
```

Comment:
I did this because in some places after 'if(use_native_clipboard())' I found calls to the 'set/getNativeBitmap' or 'setNativePalette' functions which in turn call the 'native_window_handle()' which crashed without the UI.
So to unify criteria, I used (use_native_clipboard() && App::instance()->isGui()) in every place where use_native_clipboard was used.
I'll see what I can do with this.

Reply:
I think that we can fix `native_window_handle()`:
```
  void* native_window_handle() {
    return os::instance()->defaultWindow()->nativeHandle();
  }
```
returning `nullptr` if the `defaultWindow()` is nullptr.

Reply:
Thanks David, it's the best solution.